src/client/pydaos/pydaos_shim.c

```diff
@@ -153,7 +132,7 @@ __shim_handle__err_to_str(PyObject *self, PyObject *args)
  */
 
 static PyObject *
-cont_open(int ret, char *pool, char *cont, int flags)
+cont_open(int ret, char *pool, char *cont, int ro)
```

Comment:
Can we not just pass the flag here? Changing `flags` to `ro` seems like a reduction in support

Reply:
We could, but this is the c code which is not part of the public API.  Previously flags existed but was never used and as the public interface to this is via python we should just pass in the exact options we want and do the logic checking on the python side.